spec/index.html

```diff
@@ -888,6 +889,12 @@ <h3>Triple Terms</h3>
       Every <a>triple</a> whose <a>object</a> is not a <a>triple term</a> SHOULD NOT
       use <code>http://www.w3.org/1999/02/22-rdf-syntax-ns#reifies</code> (<code>rdf:reifies</code>)
       as its <a>predicate</a>.</p>
+
+    <p>The <dfn data-lt="constituent">constituent terms</dfn> (or simply constituents)
+      of a [=triple term=] (resp. an [=RDF triple=]) are its [=subject=], its [=predicate=], its [=object=],
```

Comment:
`resp.`? This expands in my head to `respectively` (and minimally requires a following comma, in every instance), but I don't know what the `respective` reference is.

Abbreviations like this should generally be avoided, especially when, as here, their meaning might be at all confusing.